docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,561 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initialization Schemes for Input Convex Neural Network"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As input convex neural networks (ICNN) are notoriously difficult to train ([Richter-Powell et al., 2021](https://arxiv.org/pdf/2111.12187.pdf), [Bunne et al. (2022)](https://arxiv.org/pdf/2206.14262.pdf) propose to use closed-form solutions between Gaussian approximations to derive relevant parameter initializations for ICNNs.\n",
```

Comment:
left but no right parenthesis for Richter-Powell

Reply:
Done.